CHANGELOG.md

```diff
@@ -6,7 +6,8 @@
 
 ### Breaking Changes
 
-_None_
+* CoreData: Changes `fetchRequest` auto-generated method to `createFetchRequest` to avoid ambiguous function usage.  
+  [David Rothera](https://github.com/davidrothera)
```

Comment:
Please add a link to the pull request using the same format as other changelog entries.